test/JDBC/upgrade/latest/schedule

```diff
@@ -535,3 +535,4 @@ babel_4328_datetime2
 babel_4328_datetimeoffset
 babel_726
 BABEL-3401
+BABEL-3820
```

Comment:
What happened to the version prior than latest ?
We should make sure the prior created view/procedure not crash for NULL input.